cubi_tk/snappy/retrieve_irods_collection.py

```diff
@@ -110,60 +98,50 @@ def multi_assay_warning(assays):
             f"All available UUIDs:\n{multi_assay_str}"
         )
 
-    def retrieve_irods_data_objects(self, irods_path):
+    def retrieve_irods_data_objects(
+        self, irods_path: str
+    ) -> typing.Dict[str, typing.List[iRODSDataObject]]:
         """Retrieve data objects from iRODS.
 
         :param irods_path: iRODS path.
-        :type irods_path: str
 
         :return: Returns dictionary representation of iRODS collection information. Key: File name in iRODS (str);
-        Value: list of IrodsDataObject (attributes: 'file_name', 'irods_path', 'file_md5sum', 'replicas_md5sum').
+        Value: list of iRodsDataObject (attributes: 'file_name', 'irods_path', 'file_md5sum', 'replicas_md5sum').
```

Comment:
this is should be the 'native' `iRODSDataObject` now, which has a full set of attributes